index.html

```diff
@@ -37,6 +37,148 @@ <h1>Hey there. I'm <br />Subrat </h1>
         <i class="fa fa-chevron-down"></i>
       </span>
     </section>
+    <section id="Works">
+      <h1 class="sectionHeading">My Recent <br />Works </h1>
+      <div class="bar"></div>
+      <div class="card">
+        <div class="cardImage">
 
+        </div>
+        <div class="cardBody">
+          <h2 class="cardHeading">Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="cardTags">
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+          </ul>
+          <a href="#" class="cardAction">See Project</a>
+        </div>
+      </div>

```

Comment:
- [ ] I like the way that the buttons look the same as that of the template you choose. However, to perfectly fulfil the project requirement as regards `buttons or links`, I would like to advise that you make the buttons have **_hover_**, **_focus_**, and **_active_** interactions. You can find the button interaction details on the site of the Figma template you chose. It looks like this for your template:
- [ ] 
![image](https://user-images.githubusercontent.com/77106962/143510734-f63350b0-c9b9-476d-a388-37da2cdb8234.png)

Reply:
updated